src/biokbase/narrative/tests/test_jobcomm.py

```diff
@@ -260,7 +348,7 @@ def test_send_comm_msg_ok(self):
         self.jc._comm.clear_message_cache()
 
     def test_send_error_msg__JobRequest(self):
-        req = make_comm_msg("bar", "aeaeae", True)
+        req = make_job_request("bar", "aeaeae")
```

Comment:
`make_comm_msg(..., ..., True)` turns the comm message into a job request, so just used the `make_job_request` function directly.

All other instances of `make_comm_msg` had `False` as the third argument, so I changed the API of `make_comm_msg` to remove that third arg since it was redundant.